src/Shipping/Shipping.php

```diff
@@ -78,7 +77,7 @@ public function handle(): void
      */
     protected function detectCountry()
     {
-        $this->country = CountryHelper::get($returnArray = false);
+        $this->country = CountryHelper::get();
```

Comment:
Where is the CountryHelper coming from?

Reply:
I think you've created an alias so it didn't interfere with `Country` model class. This is not necessary anymore. I'll update it.